Convert some uses of passive voice to active voice in Orleans docs (#45474)
dotnet/docs: docs/orleans/resources/frequently-asked-questions.md

### Can I freely use Orleans in my project?

> Absolutely. We have released the source code under the [MIT license](https://github.com/dotnet/orleans/blob/main/LICENSE). NuGet packages are published on [nuget.org](https://www.nuget.org/profiles/Orleans).

Reviewer: ```suggestion
Absolutely. The source code is licensed under the [MIT license](https://github.com/dotnet/orleans/blob/main/LICENSE). NuGet packages are published on [nuget.org](https://www.nuget.org/profiles/Orleans).
```

### Is Orleans production ready? I heard it's a research project.

> Orleans initially started as a research project within Microsoft Research. It later grew into a product and has been used in production within Microsoft since 2011, and by other companies, after we publicly released it in 2015. Orleans is production-ready and powers many highly available systems and cloud services.

Reviewer: ```suggestion
Orleans started as a research project within Microsoft Research. It later grew into a production-ready product and has been used in production within Microsoft (since 2011) and by other companies (since it was publicly released in 2015). Orleans powers many highly available systems and cloud services.
```

Reviewer: Trying to avoid "we".

### Does Microsoft support Orleans?

> We have released the source code of Orleans under an MIT license on [GitHub](https://github.com/dotnet/orleans). Microsoft continues to invest in Orleans and accepts community contributions to the codebase.

Reviewer: ```suggestion
Microsoft released the source code of Orleans under an MIT license on [GitHub](https://github.com/dotnet/orleans). Microsoft continues to invest in Orleans and accepts community contributions to the codebase.
```
